dashboard/src/components/TableRow.tsx

```diff
@@ -17,6 +21,9 @@ export const TableRow: FC<ProjectTableRowProps> = ({
     return new Date(isoDate).toLocaleString();
   };
 
+  const { projectId } = useParams();
+  const { mutate: handleDeleteVersion } = useDeleteVersion(projectId, id);
```

Comment:
```suggestion
  const { mutate: deleteVersion } = useDeleteVersion(projectId, id);
```

Maybe that would be a better name? since its not really a handler but a function to execute it :)